src/main/java/reactor/netty/tcp/TcpServer.java

```diff
@@ -141,7 +141,9 @@ public final TcpServer bindAddress(Supplier<? extends SocketAddress> bindAddress
 	 * enriched bootstrap.
 	 *
 	 * @return a new {@link TcpServer}
+	 * @deprecated  as of 0.9.10. Use the methods exposed on {@link TcpServer} level.
```

Comment:
same as above, comment on which version to expect a removal

Reply:
This method is removed in master, so I'll add 1.0.0

Reply:
done